test/2.js

```diff
@@ -0,0 +1,15 @@
+
+const solution = require('../solutions/2.js');
+// solution = require('../yourSolution').solution;
+
+const test = (a, b, result) => {
```

Comment:
need a,b,c and result as parameters for test

Reply:
saumya solved